reminders.txt

```diff
@@ -18,3 +18,5 @@ Hey human, remember to have some family time (assuming you have a supportive fam
 Hey human, is there anything else you think I should learn to say? Let me know👇
 Hey human, remember to blink just enough to keep your eyes lubricated. Screens can be hypnotizing. 😵‍💫
 Hey human, how about listening to your favorite song 🎙️🎵
+Hey human, is the weather good today? Let's go for a walk!
+Hey human, guess what? It's AFK time! See you in 5 minutes! Have fun!
```

Comment:
I feel like this could be a bit clearer. How about:

Hey human, guess what? It's time to take a break from your screen - if you want! See you in 5 minutes, and have fun 😁